src/core.js

```diff
@@ -149,6 +168,8 @@ c3_chart_internal_fn.initWithData = function (data) {
     var $$ = this, d3 = $$.d3, config = $$.config;
     var defs, main, binding = true;
 
+    $$._axis = new _Axis($$);
```

Comment:
Can you remove _? $$.axis seems better from the uniformity point of view (other internal variables do not have prefix _).